frontend/public/components/monitoring/alerting.tsx

```diff
@@ -682,6 +674,9 @@ export const AlertsDetailsPage = withFallback(
     // eslint-disable-next-line react-hooks/exhaustive-deps
     const labels: PrometheusLabels = React.useMemo(() => alert?.labels, [labelsMemoKey]);
 
+    // eslint-disable-next-line camelcase
+    const runbookURL = alert?.annotations?.runbook_url;
```

Comment:
What URL's are going to be in this annotation? Are these links to Red Hat KCS?

Reply:
@sjstout what are docs thoughts on this. Should we / Could we have a section in docs, for these that teams could link to? thus we wouldn't need to use github links?

Reply:
Ask this question this way (with a hypothetical example): If Amazon said (in its console); hey your VM is having an issue (an alert); please follow this github link (runbook) from our github org to fix the issue how would you feel (as a customer)? 

Without some 'rules' around, what links we can use here; I am leary of this PR providing customer confidence and the user experience we are looking for.

Reply:
@sferich888 , I agree that pointing customers to github for official, supported product doc can be confusing. Would be happy to continue having discussions around adding this kind of content to the product doc so that customers are certain that the guidance provided is officially supported, and verified by QE.
cc: @TanyaShearonRH

Reply:
cc @sichvoge

Reply:
cc @brampling

Reply:
The proposed plan is to export the runbook content to the openshift documentation and alter the links accordingly at release time.
I think this issue shouldn't hold up this PR in any case. I think we want to display the runbook URLs in the console and the console doesn't really have any influence on where those URLs point.
I will talk to the doc team and propose the needed alteration in CMO. I think this here can go ahead. @sferich888 any objections?

Reply:
Hi, suggest:
1. runbook_url should be changed based on u/s and d/s deployment. Like:
 u/s : <github link>/<runbook name>
 d/s: <d/s docs link>/<runbook name>

2. Create automatic PRs for d/s based on u/s runbooks updates .
3. In CNV we already publish the runbooks as a github pages. Currently at ```http://kubevirt.io/monitoring/``` .

Reply:
yes we are hammering out a workflow for this right now. For OpenShift deployments the urls will point to the official docs, for others links to github will do.

I think this PR though should be unblocked. Iiuc this PR will display whatever link is in the actual alert. So the work needs to be done somewhere else, instead of holding up @kyoto's work for almost 6 months now.

@sferich888 Any objections against moving ahead with this PR?